src/App.css

```diff
@@ -1,38 +1,35 @@
-.App {
-  text-align: center;
+html{
+  min-height: 100%;
+}
+body{
+  margin:0;
+  padding:0;
+  font-family: sans-serif;
+  min-height: 100%;
+  /* background-color:#171334; */
```

Comment:
I think we should try to keep the commented-out code a minimum. This can cause issues for when we get confused about which person commented out which person's code.